monsters.py

```diff
@@ -514,7 +514,7 @@ def print_everything():
         'speed': print_speed,
     }
 
-    for (abbr, fn) in fnmap.iteritems():
+    for (abbr, fn) in list(fnmap.items()):
```

Comment:
Most of the `items()` changes in this commit don't actually need to be wrapped in a `list()` call. At least from what I can see and remember, order is either:
- not guaranteed anyways
- not important

When simply iterating over stuff, it's not required to know the entire data structure beforehand :)